src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java

```diff
@@ -31,7 +30,7 @@ public void addFormatters(FormatterRegistry registry) {
 
   @Override
   public void addCorsMappings(CorsRegistry registry) {
-    if (StringUtils.isNotBlank(corsMapping)) {
+    if (corsMapping.length != 0) {
       registry.addMapping("/**").allowedOrigins(corsMapping);
```

Comment:
Could you please also add the `.allowedMethods("GET", "HEAD", "POST", "PUT", "DELETE", "OPTIONS")`?
The default are only GET,HEAD,POST.

Reply:
Done